src/Locale/en.json

```
@@ -199,8 +199,13 @@
   "SORT_OPTIONS__name": "Patient name A-Z",
   "SORT_OPTIONS__review_time": "Oldest review date first",
   "SORT_OPTIONS__taken_at": "Oldest taken date first",
+  "Search_by_UHID": "Search by UHID",
+  "Search_by_emergency_contact_phone_number": "Search by Emergency Contact Phone Number",
+  "Search_by_patient_name": "Search by Patient Name",
```

use consistent casing. 
```suggestion
  "search_by_patient_name": "Search by Patient Name",
```

---

still unresolved...

---

> still unresolved...

@rithviknishad ,solved